packages/browser/src/Component/Itowns/Itowns.js

```diff
@@ -11,6 +11,12 @@ export { LayerManager } from './LayerManager/LayerManager';
 /** @class TilesManager */
 export { TilesManager } from './3DTiles/TilesManager';
 
+/**
+ * @class CityObject
+ * @class CityObjectID
+ */
+export { CityObject, CityObjectID } from './3DTiles/Model/CityObject';
```

Comment:
why expose these classes here ? since Itowns.js is not expose in the API they are no going to be expose in @ud-viz/browser API

Reply:
Yes the index.js is the right place, you can do it the way you did it but it's better practice to expose it in the index.js (@ud-viz/browser API) so it make simpler to check what can be use outside the package (for refacto purpose, or to detect unused part of the code). Furthermore if you were using a bundle import like in the example it would be not possible to access Itowns.